packages/@react-native-windows/cli/src/generator-common/index.ts

```diff
@@ -43,6 +43,22 @@ export function resolveContents(
 ): string {
   let content = fs.readFileSync(srcPath, 'utf8');
 
+  if (content.includes('\r\n')) {
+    // CRLF file, make sure multiline replacements are also CRLF
+    for (const key of Object.keys(replacements)) {
+      if (typeof replacements[key] === 'string') {
+        replacements[key] = replacements[key].replace(/([^\r]?)\n/g, '$1\r\n');
```

Comment:
if it does find a \r, then $1 will be \r, meaning it will replace it with \r\r\n ?